### PR TITLE
EZP-28479: In the content item edit view, the application does not display the help text when I hover over the input field

### PR DESCRIPTION
**Pull request by ViniTou**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28479
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/content/form_fields.html.twig [hunk 1]
```diff
@@ -82,7 +82,12 @@
     {% set attr = attr|merge({'class': (attr.class|default('') ~ ' ez-data-source__input')|trim}) %}
     {% set wrapper_attr = wrapper_attr|default({})|merge({'class': (wrapper_attr.class|default('') ~ ' ' ~ wrapper_class)|trim}) %}
 
-    <div{% with { attr: wrapper_attr } %}{{ block('attributes') }}{% endwith %}>
+    {% if fieldtype.vars.value.fieldDefinition.descriptions[fieldtype.vars.languageCode] is defined %}
```
Comment (adamwojs) [inline src/bundle/Resources/views/content/form_fields.html.twig]: For me it looks more readable after extracting `field type.vars.value.field Definition.descriptions` to the local variable:

```twig
{% set field_type_descriptions = fieldtype.vars.value.fieldDefinition.descriptions %}
{% if field_type_descriptions[fieldtype.vars.languageCode] is defined %}
    {% set wrapper_attr = wrapper_attr|merge({'title': field_type_descriptions[fieldtype.vars.languageCode]}) %}
{% endif %}
```

Comment (lserwatka) [reply src/bundle/Resources/views/content/form_fields.html.twig]: @ViniTou could you incorporate this feedback?

Comment (ViniTou) [reply src/bundle/Resources/views/content/form_fields.html.twig]: @lserwatka 
done

**Comment by micszo** [comment]
Using Article as an example everything's fine except for the RichText fields (Summary & Body).
Instead of the set descriptions tooltip is displayed for both as on screenshot below.
screen shot 2018-05-02 at 10 55 48 2

**Comment by micszo** [comment]
@ViniTou wdyt?

**Comment by ViniTou** [comment]
@micszo 
I think it is rendered in separate view, gonna check it out. It would probably require separate PR to new richTextBundle.

**Comment by ViniTou** [comment]
Ok, that comes directly from alloy editor, is there any way we can override this? @sunpietro

**Comment by sunpietro** [comment]
@ViniTou do you mean the tooltip? It might be. But I'm not sure how we can improve the styling. I'll have to investigate it, but as far as I know it might require some effort to style it.
@dew326 might have some opinions based on his actual experience with AlloyEditor.

**Comment by dew326** [comment]
@sunpietro @ViniTou @micszo  The AlloyEditor takes the `id` of the editable element: https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/views/fieldtypes/edit/ezrichtext.html.twig#L4

One way would be to change the `id`, but ids usually are not human readable. The other option is to check AlloyEditor/CKEditor and replace the method which is adding the title. 

For me in order to make this right, we need the following ticket and dig more into AlloyEditor/CKEditor.

**Comment by webhdx** [comment]
How does it relate to the wireframes?
According to those: 
image
The description should be displayed below.

**Comment by micszo** [comment]
@webhdx The way I understood the ticket the first paragraph related to the bug and the second one to a future Improvement/Story.
And also in the CS ticket this came from a custom template was provided to the customer which displays it differently.

**Comment by lserwatka** [comment]
@ViniTou could you re-base here to trigger Behat?

**Pull request merged**